src/bundles/SellerRegistration/components/DocumentsForm/DocumentsForm.js

```diff
@@ -56,11 +61,17 @@ class DocumentsForm extends BaseForm {
         {
             'label': 'Financial statement',
             'id': 'financial',
-            'description': 'Please provide an up-to-date financial statement. If you do not have this, ask your accountant for a letter confirming financial viability.',
+            'description': 'Please provide an up-to-date financial statement. If you do not have this, ask your accountant for a letter confirming financial viability. We will not accept an internal letter as proof of financial viability.',
             'expires': false
         },
         {
-            'label': 'Professional Indemnity and Public Liability Insurance',
+            'label': 'Professional Indemnity Insurance',
+            'id': 'indemnity',
+            'description': 'Your insurer can issue a certificate of currency.',
```

Comment:
Is it intended that the description for both Professional Indemnity and Public Liability be the same?